kaggle_environments/envs/lux_ai_s3/lux_ai_s3.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "lux_ai_s3",
+  "title": "Lux AI Challenge Season 3",
+  "description": "A Novel AI Programming Challenge about Lux",
+  "version": "0.0.1",
+  "agents": [2],
+  "configuration": {
+    "episodeSteps": {
+      "description": "Maximum number of steps the environment can run. Total is this number -1.",
+      "type": "integer",
+      "minimum": 2,
```

Comment:
what does this do?

Reply:
Good question, nothing, I don't expect users to modify game parameters via kaggle environments tool (they are either fixed or randomized by the underlying lux environment). 

I thought this was needed for kaggles backend though? I recall the other envs use this